test/jdk/java/awt/KeyboardFocusmanager/TypeAhead/TestDialogTypeAhead.java

```diff
@@ -93,16 +116,14 @@ public void focusGained(FocusEvent e) {
         b.addActionListener(new ActionListener() {
                 public void actionPerformed(ActionEvent e) {
                     System.err.println("B pressed");
-
+                    d.setVisible(true);
```

Comment:
I think the d.setVisible(true) will be blocked and the next "EventQueue.invokeLater" will be executed after the dialog will be closed, unlike the old code where the invokeLater was executed after dialog became visible.

Reply:
Yes, looks like that. I have reverted this change and ran the test in CI multiple times. Link in JBS